.travis.yml

```diff
@@ -3,7 +3,6 @@ language: node_js
 sudo: false
 
 node_js:
-- 12
```

Comment:
Good job, figuring this out.. But why are you taking out version version 12 from the travis file:?

Reply:
@jkalandaibm is 12 even needed? Based on Jest 29 upgrade documentation, it doesn't support node engine version 12: https://jestjs.io/blog/2022/08/25/jest-29

Reply:
You're right.. Then according to this:m https://github.com/IBM/platform-services-node-sdk/blob/main/.travis.yml#L5
If you could add 16 and 18 to it, that would be fantastic

Reply:
I will add 16 and 18

Reply:
Yes, I will add it in a different PR

Reply:
@jkalandaibm I created the second PR:
https://github.com/IBM/networking-node-sdk/pull/102
It seems the build for node_js 18 failed 😓

Reply:
The error in the [node_js v18 CI run](https://app.travis-ci.com/github/IBM/networking-node-sdk/jobs/594834990) is:
```
node: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.27' not found (required by node)
```
Based on this `nvm` github issue that I found, it seems to be a problem related with the node and not nvm
https://github.com/nvm-sh/nvm/issues/2972

@jkalandaibm